### PR TITLE
fs.sshfs v 0.6.0

### PR DESCRIPTION
**Pull request by ReimarBauer**

### CODE REVIEW

**Comment by ReimarBauer** [comment]
Argh, that skip syntax I get mostly wrong

**Comment by ReimarBauer** [comment]
found 2 valuable problems I would have excepted to get a linter hint.

**Comment by ReimarBauer** [comment]
a later version may be compatible with windows, now we have to skip that

**Comment by ReimarBauer** [comment]
Hi @dougalsutherland 
at that package I learned already many things. But the last problem I can't solve. I get always on any CI a traceback.

What can I do to fix this?

```
Fetching package metadata .............
Traceback (most recent call last):
  File "/Users/travis/miniconda/bin/conda-build-all", line 6, in <module>
    exit(main())
  File "/Users/travis/miniconda/lib/python3.6/site-packages/conda_build_all/cli.py", line 93, in main
    b.main()
  File "/Users/travis/miniconda/lib/python3.6/site-packages/conda_build_all/builder.py", line 258, in main
    recipe_metas = self.fetch_all_metas(build_config)
  File "/Users/travis/miniconda/lib/python3.6/site-packages/conda_build_all/builder.py", line 173, in fetch_all_metas
    recipe_metas = sort_dependency_order(recipe_metas, config=config)
  File "/Users/travis/miniconda/lib/python3.6/site-packages/conda_build_all/builder.py", line 122, in sort_dependency_order
    all_deps = [dep.split(' ', 1)[0] for dep in all_deps]
  File "/Users/travis/miniconda/lib/python3.6/site-packages/conda_build_all/builder.py", line 122, in <listcomp>
    all_deps = [dep.split(' ', 1)[0] for dep in all_deps]
AttributeError: 'dict' object has no attribute 'split'
The command "if [ -n "$GH_TOKEN" ]; then echo "Creating feedstocks from the recipe(s)."; git config --global user.name "Travis-CI on github.com/conda-forge/staged-recipes"; git config --global user.email "conda-forge@googlegroups.com"; source ./.CI/create_feedstocks; else echo "Building all recipes."; source ./.CI/build_all; fi" exited with 1.
Done. Your build exited with 1.
```

**Comment by ReimarBauer** [comment]
May be it is something similar to https://github.com/tensorflow/models/issues/1587

File: recipes/fs.sshfs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  sha256: 5e9f218a382c8a82c9f639d1c0e3a622ed005fed03923966621bafbfe0da12f4
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - skip: true  # [win]
```
Comment (ocefpaf) [inline recipes/fs.sshfs/meta.yaml]: This is in the wrong place and may be the culprit of the failure you are seeing.

File: recipes/fs.sshfs/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  imports:
+    - fs.sshfs
+
+about:
+  home: https://github.com/althonos/fs.sshfs
+  license: LGPL v2.1
+  license_file: '{{ environ["RECIPE_DIR"] }}/COPYING'
+  summary: 'Pyfilesystem2 over SSH using paramiko'
+  description: >
+    Usage
```
Comment (ocefpaf) [inline recipes/fs.sshfs/meta.yaml]: I strongly encourage you do make this shorter. No need to replicate docs in the description.

File: recipes/fs.sshfs/meta.yaml (hunk 1 above)
Comment (ReimarBauer) [reply recipes/fs.sshfs/meta.yaml]: seems I am blind. Thx, lets see.

File: recipes/fs.sshfs/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5e9f218a382c8a82c9f639d1c0e3a622ed005fed03923966621bafbfe0da12f4
+
+build:
+  skip: true  # [win]
+  noarch: python
```
Comment (jakirkham) [inline recipes/fs.sshfs/meta.yaml]: Can't be `noarch` if it needs selectors unfortunately. So this will need to be dropped.

**Comment by ReimarBauer** [comment]
@jakirkham @ocefpaf anything else ;) ?

File: recipes/fs.sshfs/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    - python
+    - setuptools
+
+  run:
+    - python
+    - setuptools
+    - fs
+    - six
+    - paramiko
+    - configparser  # [py27 and py34]
```
Comment (dougalsutherland) [inline recipes/fs.sshfs/meta.yaml]: This will never trigger; you probably want `[py27 or py34]`.

**Comment by ReimarBauer** [comment]
Hi @dougalsutherland 
can I get merged this?

**Comment by dougalsutherland** [comment]
Sorry for not merging earlier. I didn't because I noticed that the pypi package pins `fs` and some other dependencies, but this recipe doesn't. I'll make an issue in the feedstock.

**Pull request merged**